docs/pages/versions.js

```diff
@@ -2,13 +2,13 @@ import * as React from 'react';
 import sortedUniqBy from 'lodash/sortedUniqBy';
 import MarkdownDocs from 'docs/src/modules/components/MarkdownDocs';
 import VersionsContext from 'docs/src/pages/versions/VersionsContext';
-import { demos, docs, demoComponents } from 'docs/src/pages/versions/versions.md?@mui/markdown';
+import * as pageProps from 'docs/src/pages/versions/versions.md?@mui/markdown';
 
 export default function Page(props) {
   const { versions } = props;
   return (
     <VersionsContext.Provider value={versions}>
-      <MarkdownDocs demos={demos} docs={docs} demoComponents={demoComponents} />
+      <MarkdownDocs {...pageProps} />
```

Comment:
Would it be better with?
```suggestion
      <MarkdownDocs pageProps={pageProps} />
```

My concern is about how can we help the future developers to find where these props are coming from.

Reply:
I agree with @flaviendelangle, the `MarkdownDocs` is the only component that connects to the `{{ }}` format anyway. I don't see why we need to pass props as `pageProps`.

Reply:
The only value would be about making it a bit more explicit, spreading tends to hide the actual structure of the props that the component receives. If it doesn't resonate, then please ignore my initial comment.

Reply:
OK I misread your comment, I thought you wanted to keep today's signature.
No strong opinion between your proposal and mine then.